app/plugin/jira/webhook.go

```diff
@@ -92,7 +92,11 @@ func (w *Webhook) SlackAttachment() (*model.SlackAttachment, error) {
 	text, err := w.renderText("" +
 		"[{{.Issue.Fields.Summary}}]({{.JIRAURL}}/browse/{{.Issue.Key}})" +
 		"{{if eq .WebhookEvent \"jira:issue_created\"}}{{if ne .Issue.Fields.Description \"\"}}" +
+		"{{if len .Issue.Fields.Description | lt 3000}}" +
```

Comment:
Clearly this works, as JB has tested its functionality, but I'm reading this line as `if len(description < 3000`, which would print the ellipsis for *shorter* messages, not longer ones. What am I missing?